parachain/runtimes/gargantua/src/lib.rs

```diff
@@ -546,9 +550,57 @@ parameter_types! {
 // We allow root only to execute privileged collator selection operations.
 pub type CollatorSelectionUpdateOrigin = EnsureRoot<AccountId>;
 
+#[derive(
+	Clone, Eq, PartialEq, codec::Encode, codec::Decode, MaxEncodedLen, TypeInfo, Debug, Default,
+)]
+pub struct ReputationDeal;
+
+#[derive(
+	Decode,
+	DecodeWithMemTracking,
+	Encode,
+	MaxEncodedLen,
+	PartialEq,
+	Eq,
+	Ord,
+	PartialOrd,
+	TypeInfo,
+	Debug,
+	Clone,
+	Copy,
+	Default,
+)]
+pub enum CurrencyAdapterHoldReason {
+	#[default]
+	CollatorSelectionBond,
+	Other,
+}
+
+impl VariantCount for CurrencyAdapterHoldReason {
+	const VARIANT_COUNT: u32 = 3;
+}
+
+pub type ReputationAsset =
+	frame_support::traits::tokens::fungible::ItemOf<Assets, ReputationAssetId, AccountId32>;
+
+pub type ReputationAssetHolder =
+	frame_support::traits::tokens::fungible::ItemOf<AssetsHolder, ReputationAssetId, AccountId32>;
+
+pub type ReputationCurrency = pallet_collator_manager::currency_adapter::FungibleToCurrencyAdapter<
+	ReputationAsset,
+	ReputationAssetHolder,
+	Balance,
+	AccountId32,
+	CurrencyAdapterHoldReason,
+>;
+
+parameter_types! {
+	pub const ReputationAssetId: H256 = H256([0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,1]);
+}
+
 impl pallet_collator_selection::Config for Runtime {
 	type RuntimeEvent = RuntimeEvent;
-	type Currency = Balances;
+	type Currency = ReputationCurrency;
```

Comment:
I actually don't think we need this. Collators should bond actual bridge tokens to signal they want to be a part of the collator set.